browser/blueprints.py

```diff
@@ -173,36 +207,24 @@ def load(filename):
         return jsonify(error), 400
 
     # Initate Project entry in database
-    project = Project.create(filename, input_bucket, output_bucket, full_path, rgb)
-    state_start = timeit.default_timer()
-    state = project.state
-    current_app.logger.debug('Got state for "%s" in %s s.',
-                             filename, timeit.default_timer() - state_start)
-
+    project = Project.create(filename, input_bucket, output_bucket, full_path)
+    project.rgb = rgb
```

Comment:
Why not include `rgb` in the instantiation?

Reply:
Whether or not a project is in RGB mode does not affect it's creation, so it made sense to simplify the `create` signature and set `rgb` outside of it. The `rgb` setting is more of a cosmetic column that affects what data we display, similar to the current frame/channel/feature.

Also, as we look towards using a file loader interface, we'll likely just pass a file loader instance to `Project.create`, which would pare the signature down to `Project.create(file_loader)`.